Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -0,0 +1,337 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Shared
+import UIKit
+import Deferred
+import Storage
+import WebImage
+
+
+// MARK: -  Lifecycle
+struct ASPanelUX {
+    static let backgroundColor = UIColor(white: 1.0, alpha: 0.5)
+    static let topSitesCacheSize = 20
+    static let historySize = 10
+}
+
+class ActivityStreamPanel: UIViewController, UICollectionViewDelegate {
+    weak var homePanelDelegate: HomePanelDelegate? = nil
+    let profile: Profile
```

Comment:
Nit: Probably better if this is private. We shouldn't need to access this outside of the panel.